pages/message-sent/index.tsx

```diff
@@ -1,15 +1,21 @@
-import { BaseContainer } from "@components/Page/BaseContainer";
-import { Helmet } from "react-helmet";
-import React from "react";
 import LayoutContainer from "@components/LayoutContainer";
-import Image from "next/image";
+import React from "react";
 import getConfig from "next/config";
+import { BaseContainer } from "@components/Page/BaseContainer";
+import { BetterSocialEventTracking } from "analytics/analyticsEventTracking";
+import { Helmet } from "react-helmet";
+import { sendAnalytics } from "@services/analytics/analyticsServices";
 import { useRouter } from "next/router";
 
 const { publicRuntimeConfig } = getConfig()
 
 export default function MessageSent() {
     const router = useRouter();
+
+    React.useEffect(() => {
+        sendAnalytics(BetterSocialEventTracking.SUCCESS_SCREEN_OPEN)
+    }, [])
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useEffect` hook is used to send analytics when the component mounts. However, it's important to handle any potential errors that might occur during this process. If the `sendAnalytics` function fails and throws an error, it could cause the entire component to unmount and disrupt the user experience. Consider adding error handling within the `useEffect` hook.
```diff
-    React.useEffect(() => {
-        sendAnalytics(BetterSocialEventTracking.SUCCESS_SCREEN_OPEN)
-    }, [])
+    React.useEffect(() => {
+        sendAnalytics(BetterSocialEventTracking.SUCCESS_SCREEN_OPEN).catch(error => {
+            console.error('Failed to send analytics:', error);
+        });
+    }, [])
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->